caas/kubernetes/provider/k8swatcher.go

```diff
@@ -46,43 +48,55 @@ func newKubernetesNotifyWatcher(wi watch.Interface, name string, clock jujuclock
 
 const sendDelay = 1 * time.Second
 
+func (w *kubernetesNotifyWatcher) OnAdd(obj interface{}) {
```

Comment:
These need comments, usually we say what Interface the method comes from, eg
// OnAdd implements FooInterface